qt/python/mantidqt/widgets/instrumentview/api.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Could you also chmod the file? Github still thinks it is executable.

Reply:
done